readme.md

```diff
@@ -83,18 +83,18 @@ The hash of each rev'd file is stored at `file.revHash`. You can use this for cu
 ### Asset manifest
 
 ```js
-const gulp = require('gulp');
-const rev = require('gulp-rev');
+import {src, dest} from 'gulp';
```

Comment:
I would prefer to keep `gulp` a default import.

Reply:
➡️ https://github.com/sindresorhus/gulp-rev/pull/267/commits/39e09fa8de35869791c9942f33c444927079a1c7